### ISSUE TITLE
Hats for mmi bots

### ISSUE THREAD

**Issue by Midaychi** [d3athrow/vgstation13#5235]
If #5000 gets merged, we'll have a serious problem that needs to be addressed: The lack of hats for them!

So, it would be super neat if there were a way to equip hats on them and have them able to equip hats on their own. Using click-drag or something. Since their icon shapes don't change too much, it should be relatively easy to position where the hats should appear.

Heck you could even make them equip-able on bots without mmis with this.

**Comment by FudgePucker** [comment]
Now mmi spider bots and mommis will fight for silly hats

**Comment by Midaychi** [comment]
No no. Not spider bots, though they certainly can wear them too if they want.

I meant for mulebots/medbots/floorbots/buttbots/cleanbots/etc

5000 would let you be able to put mmis in them.

**Comment by AutismSpider** [comment]
Paint job for mule-bots would look better.

Paint-job/Spoilers for the mule bot etc.

Butt/clean/med etc should be hat and pain job for it.

**Comment by Midaychi** [comment]
Could be neat, but you'd need overlays and a spray machine in the mech bay for it.

**Comment by AutismSpider** [comment]
Give the god damn mech bay some use if the mmi bots  walk on the port and hud pops up for spray machine overlay stuff etc.

**Comment by Midaychi** [comment]
I went through and split the cleanbot, floorbot, medbot, buttbot, mulebot, and the ED into mostly greyscaled overlay parts and added hatches.

https://dl.dropbox.com/s/48nbzg4uzynry3d/aibotsgrey.dmi

Didn't touch the other bots cause they're hardly used if at all, or are too complex for a non-artist like me to want to bother greyscaling/splitting up (like the securtron. It's got light refraction and specular *baked in*. Fuck that.)
I also did attempt to add light to the above bots that didn't have them, though. Probably looks like ass.

They're split into _chrome (the framework), _light (the light), _hatch (the hatch), _body (The part that'd probably get colourized).  There's also other bits and bobs that I thought relevant, some greyscale and some not, all split out. Stuff that ends in _ani is animated.
